### PR TITLE
Updating the README

### PR DESCRIPTION
**Pull request by eviau-artefactual**
Hello!

A surprise PR with a few updates to the README for clarification, with the goal of helping contributors find the information they need.
[
View how it would look like in the end](https://github.com/eviau-artefactual/archivematica/blob/00d7dbe1abec44f6bfac51678c2bd5b275084d16/README.md)

Let me know what you think?

### CODE REVIEW

**Comment by eviau-artefactual** [comment]
If you want to know: the picture for the repo was taken from the current website... Maybe we have another one, or maybe the logo would be best?

File: README.md [hunk 1]
```diff
@@ -5,44 +5,49 @@
...
-* [Wiki](https://www.archivematica.org/wiki/Development): Developer facing documentation, requirements analysis and community resources
-* [Issues](https://github.com/archivematica/Issues): Git repository used for tracking Archivematica issues and feature/enhancement ideas
+* [Slideshare repo](https://fr.slideshare.net/Archivematica/presentations) and [YouTube channel](https://www.youtube.com/@ArtefactualSystems) for previous trainings and talks
 * [User Google Group](https://groups.google.com/forum/#!forum/archivematica): Forum/mailing list for user questions (both technical and end-user)
 * [Paid support](https://www.artefactual.com/services/): Paid support, hosting, training, consulting and software development contracts from Artefactual
 
+## Developing with Archivematica
+
+* [Archivematica API documentation](https://archivematica.org/fr/docs/archivematica-1.15/dev-manual/api/api-reference-archivematica/#api-reference-archivematica): Get to know the Archivematica API
+* [Wiki](https://wiki.archivematica.org/Development): Developer facing documentation, requirements analysis and community resources
```
Comment (replaceafill) [inline README.md]: New developers usually set up AM for development using the README file in the hack directory nowadays. We don't maintain the Wiki anymore so I'd recommend removing this link. If you find anything that is still relevant there we should incorporate in the README of the hack directory instead. GitHub wikis might be an alternative too.

**Comment by replaceafill** [comment]
> If you want to know: the picture for the repo was taken from the current website... Maybe we have another one, or maybe the logo would be best?

The logo would be my choice, but I'd like to hear @sromkey and @Dhwaniartefact opinion on this.

**Comment by sromkey** [comment]
> > If you want to know: the picture for the repo was taken from the current website... Maybe we have another one, or maybe the logo would be best?
> 
> The logo would be my choice, but I'd like to hear @sromkey and @Dhwaniartefact opinion on this.

Sure, the logo sounds like a good choice!

**Comment by eviau-artefactual** [comment]
My only constraint is having a logo with a background that is not transparent - or something else that makes it visible both on a dark theme and on a light theme... ?

**Comment by eviau-artefactual** [comment]
Ready for second review - if it works for you we can add the logo in a potential other PR in the future :)

File: demo_s004.png
Comment (sromkey) [inline demo_s004.png:1]: As we discussed in our meeting, maybe the screenshot should be removed because it's hard to keep up to date.

**Review by sromkey — APPROVED** [review]
looks good @eviau-artefactual !

File: README.md [hunk 2]
```diff
@@ -72,14 +88,21 @@ For more projects in the Archivematica ecosystem, see the [getting started] page
...
-[Archivematica Issues repository]: https://github.com/archivematica/Issues/issues
 [SECURITY file]: SECURITY.md
 [Archivematica GitHub]: https://github.com/artefactual/archivematica
 [Storage Service]: https://github.com/artefactual/archivematica-storage-service
 [Format Policy Registry]: https://github.com/artefactual/archivematica/tree/qa/1.x/src/dashboard/src/fpr
 [getting started]: https://wiki.archivematica.org/Getting_started#Projects
+[Website]: https://www.archivematica.org/
+[Slideshare repo]: https://fr.slideshare.net/Archivematica/presentations
+[YouTube channel]: https://www.youtube.com/@ArtefactualSystems
+[Archivematica API documentation]: https://archivematica.org/fr/docs/archivematica-1.15/dev-manual/api/api-reference-archivematica/#api-reference-archivematica
```
Comment (eviau-artefactual) [inline README.md]:
```suggestion
[Archivematica API documentation]: https://archivematica.org/docs/archivematica-1.16/dev-manual/api/api-reference-archivematica/#api-reference-archivematica
```

File: README.md [hunk 3]
```diff
@@ -72,14 +88,21 @@ For more projects in the Archivematica ecosystem, see the [getting started] page
...
 [Paid support]: https://www.artefactual.com/services/
 [contributing guidelines]: CONTRIBUTING.md
-[Archivematica Issues repository]: https://github.com/archivematica/Issues/issues
 [SECURITY file]: SECURITY.md
 [Archivematica GitHub]: https://github.com/artefactual/archivematica
 [Storage Service]: https://github.com/artefactual/archivematica-storage-service
 [Format Policy Registry]: https://github.com/artefactual/archivematica/tree/qa/1.x/src/dashboard/src/fpr
 [getting started]: https://wiki.archivematica.org/Getting_started#Projects
+[Website]: https://www.archivematica.org/
+[Slideshare repo]: https://fr.slideshare.net/Archivematica/presentations
```
Comment (eviau-artefactual) [inline README.md]:
```suggestion
[Slideshare repo]: https://slideshare.net/Archivematica/presentations
```

**Pull request merged**